server_manager/www/digitalocean_account.ts

```diff
@@ -16,8 +16,8 @@ import * as crypto from '@outline/infrastructure/crypto';
 
 import {DigitalOceanServer} from './digitalocean_server';
 import {getShellExportCommands, ShadowboxSettings} from './server_install';
+import * as do_install_script from '../../output/build/server_manager/install_scripts/do_install_script';
```

Comment:
Pointing to the output directory is not a good practice. It makes the code hard to port and makes the output location impossible to change.

Can we tell the compiler that the output is another source root? Or perhaps we introduce a package with the generated code?

We currently use symbolic links to the output because the multiple roots wasn't supported at the time and it was hard to figure it out. It also helped make it work with the IDE.

/cc @sbruens

Reply:
There are different possibilities. We can also consider a `output/genfiles` that we add to the root in the root tsconfig, and put all generated files there. That might be easiest.

Reply:
Honestly, I don't think we have a problem to fix. Why don't you leave the generated files like they are for now? This has had zero impact on our productivity.

Reply:
Pulling bad builds has been a problem for us. We need to make sure our clean is robust, and we are not accidentally pulling previously built artifacts, especially on releases, but also for development, which wastes hours of debugging.

But it's all a trade-off. I think we are spending more time than it's worth it to fix something that is not a problem with the generated files.

On the other hand, Cordova puts a bunch of files and directories in the middle of our code. That is confusing for development and causes issues. Recently I had to fix the setup script because it was no longer cleaning up the directories properly.